src/audio/kpb.c

```diff
@@ -358,9 +358,111 @@ static int kpb_cmd(struct comp_dev *dev, int cmd, void *data,
 	return ret;
 }
 
+/**
+ * kpb_begin_drainning() - drain internal buffer into client's
+ * sink buffer.
+ *
+ * @arg1:  kpb component.
+ * @arg2:  clients id
+ *
+ * Return: integer representing either 0 - success
+ * or -EINVAL - failure.
+ */
 static int kpb_begin_drainning(struct comp_dev *dev, uint8_t client_id)
```

Comment:
s/kpb_begin_drainning/kpb_begin_draining

Reply:
Yes, thanks for spotting it.